bika/lims/setuphandlers.py

```diff
@@ -420,7 +420,6 @@ def addColumn(cat, col):
         addIndex(bsc, 'getCategoryTitle', 'FieldIndex')
         addIndex(bsc, 'getCategoryUID', 'FieldIndex')
         addIndex(bsc, 'getClientUID', 'FieldIndex')
-        addIndex(bsc, 'getDepartmentTitle', 'FieldIndex')
```

Comment:
@xispa , should we add something to upgrade step to remove this Index and Column from Bika Setup Catalog? I checked it and that index or column is not used anywhere.

Reply:
Be careful with removing this index. Remember that Analysis **Services** are stored in bika_setup_catalog (to which this index is added too) so you'll probably need to keep this index because of:

https://github.com/naralabs/bika.lims/blob/master/bika/lims/content/abstractbaseanalysis.py#L1011

Re your question about how to remove indexes. If an index is no longer needed, you should remove it in the upgrade step. `bika.lims.upgrade.utils.py` contains functions for doing it, see:

https://github.com/naralabs/bika.lims/blob/master/bika/lims/upgrade/utils.py#L244

If a part from the index there was a metadata column, you could use this one, instead:

https://github.com/naralabs/bika.lims/blob/master/bika/lims/upgrade/utils.py#L228